NEWS.md

```diff
@@ -47,6 +68,11 @@ The feed template for the URLHaus feed contained a spelling error:
 The correct name for the parameter "delimeter" is "delimiter". Please fix your configured bots.
 The `intelmqctl upgrade-config` command automatically fixes a configuration if the misspelling is detected.
 
+#### IntelMQ-Manager -> IntelMQ-API
+`IntelMQ-Manager` PHP backend is getting replaced by newly developed `IntelMQ-API`. The `IntelMQ-Manager` release only contains the files for web frontend. Install the `IntelMQ-API`, in order to upgrade your local installation. Follow the instructions in our [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html) to install `IntelMQ-API` & setup authentication. Unfortunately you can't keep current authentication users, you have to add users as described in the [documentation](https://intelmq.readthedocs.io/en/latest/user/intelmq-api.html#adding-a-user).
+
+**NOTE** If you're using deb/rpm packages, `IntelMQ-API` will automatically be installed if you're upgrading `IntelMQ-Manager`.
+
```

Comment:
Thats also true for the Python packages, but the rephrasing above would include this information.

Reply:
For the manager, there hasn't been a python package before.